Dockerfile

```diff
@@ -57,7 +57,7 @@ RUN set -x \
 	&& rm -rf "$GOPATH"
 
 # Install conmon
-ENV CONMON_COMMIT 6f3572558b97bc60dd8f8c7f0807748e6ce2c440
+ENV CONMON_COMMIT 65fe0226d85b69fc9e527e376795c9791199153d
```

Comment:
This is fine for now, but consider changing this over  to the tag for the release like 'v2.0.2' then below I think (didn't try it) you can do `git checkout tags/CONMON_COMMIT`, at least that's the theory.

Reply:
I think actually the next step is to enable pulling conmon from a package (like the other vms do) instead of by git commit